<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Username_label_reg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username: </string>
   </property>
  </widget>
  <widget class="QLabel" name="Password_label_reg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password: </string>
   </property>
  </widget>
  <widget class="QLabel" name="Password2_label_reg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Re-type Password: </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="DOB_groupBox_reg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>271</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Date of Birth</string>
   </property>
   <widget class="QComboBox" name="Month_comboBox_reg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select Month</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>January</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>February</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>March</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>April</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>May</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>June</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>July</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>August</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>September</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>October</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>November</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>December</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="Day_comboBox_reg">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select Day</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>26</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>27</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>28</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>29</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>31</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="Year_comboBox_reg">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select Year</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2001</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2002</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2003</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2004</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2005</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2006</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2007</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2008</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2009</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2010</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2011</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2012</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2013</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2014</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2015</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2016</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2017</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2018</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2019</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2020</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2021</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2022</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2023</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Month_label_reg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>37</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Month</string>
    </property>
   </widget>
   <widget class="QLabel" name="Day_label_reg">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>37</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Day</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>37</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Gender_groupBox_reg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Gender</string>
   </property>
   <widget class="QRadioButton" name="Male_radioButton_reg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Female_radioButton_reg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>69</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="AccountType_groupBox_reg">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Account Type</string>
   </property>
   <widget class="QRadioButton" name="User_radioButton_reg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>69</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Admin_radioButton_reg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>69</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Genres_groupBox_reg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Favourite Genres</string>
   </property>
   <widget class="QCheckBox" name="Action_checkBox_reg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Comedy_checkBox_reg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Comedy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Romance_checkBox_reg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Romance</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Drama_checkBox_reg">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Drama</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Horrour_checkBox_reg">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Horrour</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Other_checkBox_reg">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Register_pushButton_reg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Username_lineEdit_reg">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>113</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password_lineEdit_reg">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>113</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordRetype_lineEdit_reg">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>113</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Password_Error_label_reg">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password does not match</string>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordRetype_Error_label_reg">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password does not match</string>
   </property>
  </widget>
  <widget class="QLabel" name="Age_Error_label_reg">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>You are too young</string>
   </property>
  </widget>
  <widget class="QLabel" name="Username_Error_label_reg">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username already exists</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
